src/components/page.jsx

```diff
@@ -46,12 +46,21 @@ class Page extends React.Component {
       }
       this.props.actions.saveNotebook(true)
     }, 1000 * 60)
+
+    this.getPageWidth = this.getPageWidth.bind(this)
   }
 
   shouldComponentUpdate(nextProps) {
     return !deepEqual(this.props, nextProps)
   }
 
+  getPageWidth() {
+    let width = '100%'
+    if (this.props.viewMode === 'presentation') width = 'unset'
```

Comment:
Oddly, the `unset` keyword actually keeps the width at `100%` in presentation mode. Could you replace this with `undefined` which will actually skip doing anything with the `width` property for this element?

Reply:
Strange! `undefined` was not working as expected while I was working on this issue whereas `unset` was. I'll just check if there is something loose on my end and send a fix.